projects/distributed-tracing/src/pages/trace-detail/trace-detail.page.component.ts

```diff
@@ -43,6 +43,13 @@ import { TraceDetails, TraceDetailService } from './trace-detail.service';
           <div class="separation"></div>
 
           <ht-copy-shareable-link-to-clipboard class="share"></ht-copy-shareable-link-to-clipboard>
+
+          <ht-download-json
+            class="download"
+            [dataSource]="this.exportSpans$"
+            [fileName]="traceDetails.id"
```

Comment:
a couple questions on the api now that I see it in action:
1. should fileName include extension? It doesn't look like it is based on the args here, but I would expect it to
2. why take tooltip in as an input rather than letting the parent add a tooltip on directly? (via `htTooltip`)? Usually the only reason for that is if the tooltip should only apply to a section of the component, but don't think that's the case here.

Reply:
Done!!